src/components/kubectl/dashboard.ts

```diff
@@ -30,9 +32,11 @@ let terminal: vscode.Terminal;
  * @returns Boolean identifying if we think this is an AKS cluster.
  */
 async function isAKSCluster (): Promise<boolean> {
-    const nodes =  await kubectl.invokeAsync('get nodes -o json');
-    const nodesJson = JSON.parse(nodes.stdout);
-    const nodeItems = nodesJson.items;
+    const nodes = await kubectl.asJson<KubernetesCollection<Node>>('get nodes -o json');
```

Comment:
wondering if we should have `asJson` append the `-o json` so somebody down the line won't accidentally forget

Reply:
Yeah... I thought about that, but then I thought "but what if someone already passed `-o json`" and then "what if someone messes up and passes `-o yaml`, what sort of confusing error message will they get then" and it felt like it might turn into a pit of edge cases.

And then just now I discovered that `kubectl` _doesn't care_ if you pass multiple output options, it just uses the last one, so PARTY TIME.

Except wait, what if the command has a trailing argument (a la `--`) so you can't stick extra args on the end... *sigh*